src/app-shared-elements/gdgspain-animate-page.html

```diff
@@ -11,6 +11,9 @@
 <link rel="import" href="../../bower_components/neon-animation/neon-animation-runner-behavior.html">
 
 <script>
+  /**
+   * @polymerBehavior Polymer.GDGSpainAnimatePageImpl
```

Comment:
This should not be without `Impl`?

`* @polymerBehavior Polymer.GDGSpainAnimatePage`